src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Telemetry/VsTelemetryService.cs

```diff
@@ -9,15 +9,26 @@ namespace Microsoft.VisualStudio.ProjectSystem.VS.Telemetry
     [Export(typeof(ITelemetryService))]
     internal class VsTelemetryService : ITelemetryService
     {
-        public void PostEvent(TelemetryEvent telemetryEvent)
+        const string EventPrefix = "vs/projectsystem/";
+
+        public void PostEvent(string telemetryEvent)
+        {
+            TelemetryService.DefaultSession.PostEvent($"{EventPrefix}{telemetryEvent}");
+        }
+
+        public TelemetryEventCorrelation PostOperation(string operationPath, TelemetryResult result, string resultSummary = null, TelemetryEventCorrelation[] correlatedWith = null)
         {
-            TelemetryService.DefaultSession.PostEvent(telemetryEvent);
+            return TelemetryService.DefaultSession.PostOperation(
```

Comment:
I saw that, but we're not concerned about length here, just whether or not the reload succeeds, and why if not.